### PR TITLE
Bluetooth: Mesh: Run mesh with SoftDevice Controller

### PR DESCRIPTION
**Pull request by PavelVPV**

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -101,7 +101,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: de87b6a5ee7a80cd1b6e0a549f896ba7f921bb59
+      revision: pull/475/head
```
Comment (Balaklaka) [inline west.yml:104]: We should probably not use this revision, but wait for the revision to comming from nrfxlib.

Comment (joerchan) [reply west.yml:104]: Correct, this is temporary to run CI before nrfxlib is merged. And then this can be updated.

**Review by Balaklaka — APPROVED** [review]

**Review by joerchan — APPROVED** [review]

**Review by trond-snekvik — CHANGES_REQUESTED** [review]
conflicts with manifest changes now, otherwise good

**Review by mia-ko — APPROVED** [review]

**Review by trond-snekvik — APPROVED** [review]

**Review by mia-ko — COMMENTED** [review]
Please remember to update ncs\nrf\doc\nrf\ug_bt_mesh_configuring.rst as well before merging :)

**Review by tejlmand — APPROVED** [review]

**Pull request merged**